README.md

```diff
@@ -1094,6 +1095,89 @@ Server: spray-can/1.3.3
 }
 ```
 
+## GET /workflows/:version/:id/call-metadata
+
+This endpoint returns a superset of the data from #get-workflowsversionidlogs in essentially the same format.
+In addition to stdout and stderr this also returns inputs, start and end date, backend-specific job id, and
+return code.
+  
+### Notes
+
+- The logs endpoints could possibly be deleted once this is implemented, nobody is calling those anyway (but
+  check with Chet first).
+  
+- It seems strange that callers want inputs to calls but not outputs (other than stdout and stderr).
```

Comment:
It does say that doesn't it.  Well that's even weirder, we don't report non-file outputs?